axis_backend/apps/kpis/services/kpi_type_service.py

```
@@ -0,0 +1,226 @@
+        )
+
+    # Update Operations
+
+    @transaction.atomic
+    def update_kpi_type(
+        self,
+        kpi_type_id: str,
+        name: Optional[str] = None,
+        description: Optional[str] = None,
+        weight: Optional[int] = None,
+        **kwargs
```

Ruff (0.14.5)

Missing type annotation for `**kwargs`

(ANN003)